IFC 4.3/SpatialStructure_2/readme.md

```diff
@@ -0,0 +1,29 @@
+
+## SpatialStructure_2
+
+### Intent
+
+This UT establishes a simple spatial structure of a bridge.
+
+It is based on the storyline [IR\_SLBH-AM](https://app.box.com/file/719326006551?s=avj19mygfavc9035ankkfendwik6n5n5).
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+- ProjectSetup_1
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+- spatial structure
+  - `IfcSite`
+  - `IfcBridge`
+  - `IfcBridgePart`
```

Comment:
IfcBridgePart is deprecated. Please use IfcFacilityPart instead.

Reply:
Resolved in 41b988f .